adobe/spectrum-css
# fix(asset list + miller columns): selected/hover/navigated color contrast

## Description

- This fixes the issue where asset list and miller columns display white text on a light gray background by setting the value to HighlightText rather than SelectedItemText.

## How and where has this been tested?

- Verified in assistiv labs Windows High Contrast Mode

### Validation steps

1. Run Storybook locally (or reference the link for this PR).
2. Navigate to Assistiv Labs and start a Windows High Contrast Mode VM (if running the branch locally, you'll need to enable the Assistiv Labs tunnel to access Storybook).
3. Enable Windows High Contrast Mode.
4. Navigate to the Asset List and Miller Columns components in Storybook.
5. Verify that the selected/hover/navigated states display black text on the appropriate backgrounds.

### Regression testing

Validate:

1. The documentation pages for at least two other components are still loading, including:


2. If components have been modified, VRTs have been run on this branch:


## Screenshots

<img width="495" alt="Screenshot 2024-09-10 at 12 28 51 PM" src="https://github.com/user-attachments/assets/e4896d82-d112-432c-b807-06c047ab9c75">
<img width="617" alt="Screenshot 2024-09-10 at 12 50 41 PM" src="https://github.com/user-attachments/assets/26244ae1-c704-48f6-9f57-4dc1da6645d0">
<img width="580" alt="Screenshot 2024-09-10 at 12 50 37 PM" src="https://github.com/user-attachments/assets/542e5666-dc20-4aa4-ac3d-60c07498de52">

## To-do list
